assets/package-list.json

```diff
@@ -385,7 +385,15 @@
                 "requirements": {"go":"1.12.0"}
             },
 
-
+            {
+                "title": "Test Center",
+                "name": "test-center",
+                "version": "0.1.0",
+                "url": "https://github.com/akamai/cli-test-center",
+                "issues": "https://github.com/akamai/cli-test-center/issues",
+                "commands": [{"name":"test-center","version":"0.1.0","description":"Test Center is a testing tool that checks the effect of configuration changes on your web property. Use this tool as part of your testing protocol to increase your confidence in the safety and accuracy of your configuration changes."}],
+                "requirements": {"go":"1.15"}
```

Comment:
I guess that, being a node.js CLI package, this is not really correct

Reply:
You also need to change your cli.json or your CLI package will be unusable via akamai CLI https://github.com/akamai/cli-test-center/blob/master/cli.json

Reply:
We are pushing go CLI Package.

Reply:
Understood @rakayada, my apologies. As I told you directly some minutes ago, this was just a misunderstanding from my side.